apps/platform-app/src/utils/validateCvFileHelper.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from 'zod'
+
+export const cvFileSchema = z.object({
+  name: z.string(),
+  type: z.literal('application/pdf'),
+  size: z.number().max(4.5 * 1024 * 1024), // 4.5 MB
```

Comment:
**suggestion**: Consider increasing the maximum file size limit if larger CV files are expected.